iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ShuntCompensatorImpl.java

```diff
@@ -47,13 +39,12 @@
     private final TDoubleArrayList targetDeadband;
 
     ShuntCompensatorImpl(Ref<? extends VariantManagerHolder> network,
-                         String id, String name, boolean fictitious, double bPerSection, int maximumSectionCount,
+                         String id, String name, boolean fictitious, ShuntCompensatorModelWrapper model,
                          int currentSectionCount, TerminalExt regulatingTerminal, boolean voltageRegulatorOn,
                          double targetV, double targetDeadband) {
         super(id, name, fictitious);
         this.network = network;
-        this.bPerSection = bPerSection;
-        this.maximumSectionCount = maximumSectionCount;
+        this.model = model;
```

Comment:
Attach the model this this shunt compensator, like it's done between HvdcLine and HvdcConverterStation